### ISSUE TITLE
The information biometric screen for passwords can be accessed only one time 

### ISSUE THREAD

**Issue by abodea** [mozilla-mobile/firefox-ios#15509]
## Steps to reproduce
1. Launch Firefox.
2. Access the password section from the hamburger menu or settings.
3. Tap on the cancel button.
4. Revisit the password section.
### Expected behavior
The biometric screen with all the details about data protection: `Your passwords are now protected by Face ID, Touch ID or a device passcode.`
### Actual behavior
The biometric screen for passwords can be accessed only once.
### Device & build information
* Device: iPhone 13 Pro (15.7.1) 
* Build version: v9000 ( 
* First seen version: N/A 
### Notes
Please note that even if leaving the passwords section by tapping on the cancel button, it counts as tapping on continue, and the next visit to the password section the Face ID access notification prompt is displayed. 
Attachments: 

https://github.com/mozilla-mobile/firefox-ios/assets/42831109/48895d4f-aea4-45b7-abbc-4e915a9b69fc

**Comment by abodea** [comment]
![image](https://github.com/mozilla-mobile/firefox-ios/assets/42831109/ce876dca-b092-4219-b7ef-6e5e0523be49)

Please note that this screen should be displayed until the user press on the continue button, no matter from where the password section was accessed hamburger menu, or settings.

**Comment by lmarceau** [comment]
This could relate to the settings coordinator changes. I'll have a look.

**Comment by isabelrios** [comment]
There is another issue with this screen... first time accessing logins from settings, sometimes the locked screen is shown briefly before the biometric screen that should appear the first time. Please see video with the issue:


https://github.com/mozilla-mobile/firefox-ios/assets/1897507/3d07019f-a96d-443a-bead-a5f463ee5df5


This is affecting one of the automated tests in the smoketest4, it fails the first time but passes in the second retry because the first screen with the Continue button is not shown, while in the first case it is shown but late, when the test can not click on the Continue button to go to the unlock screen.

**Comment by lmarceau** [comment]
@abodea this behavior is something we had in previous application as well I believe. I tested in v113 Testflight (to ensure there's no coordinator changes affecting this) and this is what I get (that screen is only shown once).

https://github.com/mozilla-mobile/firefox-ios/assets/11338480/6bbe88d0-2487-4b39-bd54-84f3cdd2f0ba

@isabelrios I'll look into this other issue.

**Comment by abodea** [comment]
Yes, this is happening in v116 but it's the wrong behavior because I didn't tap on the `Continue` button, and still somehow when accessing for the 2nd time password section, is bypassed, and the user is not informed about anything. 
So it feels like the `Continue` button was pressed but in reality, it wasn't.

**Comment by dnarcese** [comment]
If this is not a regression, it is not high priority right now.

**Comment by lmarceau** [comment]
@isabelrios for the `eng:bug-auto-found` problem you mentioned I think we should open a different issue, since I don't think this relates to the original task mentioned here 🙏

**Comment by isabelrios** [comment]
> @isabelrios for the `eng:bug-auto-found` problem you mentioned I think we should open a different issue, since I don't think this relates to the original task mentioned here 🙏

Sure, I can open a different issue. I do think there is a regression unless the locked screen and the one with the Continue button have changed when they have to appear. This test has been passing for a long time and now it is not due to the order of the screens and how they are shown differently when accessing Logins menu from different places (Setting Menu and Browser Tab Menu).

**Comment by data-sync-user** [comment]
➤ Stefan Vladut commented:

I couldn’t reproduce this issue anymore. I’m sure it was happening in the past because I reproduced it many times but it might have been solved with the changes on the settings navigation flow

**Comment by data-sync-user** [comment]
➤ Andrei Bodea commented:

Hi, I can still reproduce this issue on v127 (41813) with iPhone 13 Pro (15.7.1).

!RPReplay_Final1715863670.mov|width=33.33333333333333%,alt="RPReplay_Final1715863670.mov"!
